include/ck/tensor_operation/gpu/device/impl/device_grouped_conv_bwd_data_multiple_d_xdl_cshuffle_v1.hpp

```diff
@@ -239,6 +269,8 @@ struct DeviceGroupedConvBwdDataMultipleD_Xdl_CShuffle_v1
     static_assert(NDimSpatial == 2 || NDimSpatial == 3,
                   "wrong! only implemented for 2D and 3D now");
 
+    static constexpr index_t MaxKernelArgsNum = 32;
```

Comment:
Please add some doc.
```suggestion
    static constexpr index_t MaxGroupedGemmGroupsNum = 32;
```